common/deposits/hfe_deposit.txt

```diff
@@ -346,4 +367,211 @@ hfe_deposit_memorial_4_0 = {
 			has_deposit = hfe_deposit_memorial_4_0
 		}
 	}
-}
\ No newline at end of file
+}
+
+###################################################
+
+#	VARELSE		#
+
+hfe_deposit_mechanicus_farm_spire = {
+	icon = d_broken_spire
+	is_for_colonizeable = yes
+	category = deposit_cat_food
+	use_for_min_max_adjustments = no
+	use_weights_for_terraforming_swap_types = no
+	should_swap_deposit_on_terraforming = no
+
+	potential = {
+		planet = {
+			OR = {
```

Comment:
i think deposits generated before planet flag set

Reply:
Not true. This Deposit spawns correctly.